Quotas.qmd

```diff
@@ -0,0 +1,643 @@
+---
+title: Quotas and Limitations
+---
+```{=html}
+
+<h2>Copernicus General Users </h2>
+<thead>General Users are all users registered on Copernicus Data Space Ecosystem on the basis of self registration and not falling into the user typologies defined in the subsequent sections. There are no restrictions on who can register to access the data. </thead> <br><br>
+
+<div class="table-responsive">
+<table class="table-roadmap">
+  <tr>
+    <th style="width:6%">#</th>
+    <th>Limitations/Quotas</th>
+    <th>Sentinel Hub APIs⁸</th>
+    <th>Data Workspace API</th>
+    <th>OData, STAC, S3</th>
+    <th>Direct HTTP access to COGs</th>
+    <th>openEO API / Algorithm plaza</th>
```

Comment:
As discussed last time, I think we should show a distinction of a header with the rest of the table. 

Though I would still prefer the CDSE theme, if you think it is not necessary here, I would still suggest including either a horizontal line after header and making headers bold

Reply:
OK then I also prefer applying the usual CDSE theme, I'll implement it in my upcoming commit.